### ISSUE TITLE
hack (fn) RAM Cost when accessing ns.gang.getAscensionResult().hack

### ISSUE THREAD

**Issue by barri** [bitburner-official/bitburner-src#301]
I have a script which automates the gang part of the game. This script uses the ns.gang.getAscensionResult(gangMember) function.
When storing the return object in a variable and accessing the hack property of it the RAM calculator interprets this as a call to the hack function and thus penalizes the script with a 100 MB ram cost for the (non-existing) function call.

Code to reproduce:
```
    for (let m of ns.gang.getMemberNames()) {
        let ascResults = ns.gang.getAscensionResult(m);
        if (ascResults.hack > 0) {
            ns.tprint("foo");
        }
    }
```
Version: 2.2.1 (c46cedd5)

**Comment by Snarling** [comment]
To fix this we will need to rename that property to "hacking" which will technically be a small API break. Will fix in 2.3

For now you can avoid the ram cost by referring to it as `ascResults["hack"]`

**Comment by Snarling** [comment]
Duplicate of #298 , closing this one
